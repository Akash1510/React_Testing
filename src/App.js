
import './App.css';


function App() {
  return (
    <div className="App">
   <p>Hey I Am Akash</p>
   <p>Not to Anything</p>
   <input type='text' placeholder='Enter your Name' name='Doctor' id='name' className='local'/>
   <br/>
   <br/>
   <br/>
   <img title='this is leaf' src='https://th.bing.com/th/id/OIP.MPP1fw6m0Gy1xNGZF2ioIAHaFG?rs=1&pid=ImgDetMain' alt='prompt.jpg'/>

    </div>
  );
}

export default App;
