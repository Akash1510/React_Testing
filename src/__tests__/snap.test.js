import { render } from "@testing-library/react";
import App from "../App";

test('Checking of the Snapshots', () => { 
    
   const view = render(<App/>)
   expect(view).toMatchSnapshot();
 })

//   If make changes in the App.js files at last of the Production 
// then the snapshots use 
// snapshots detect the Pyq code and Recent code its does not pass
// if you want to update then you can using the U 
// otherwiese not
