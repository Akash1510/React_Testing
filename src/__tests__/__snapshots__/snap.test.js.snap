// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking of the Snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <p>
          Hey I Am Akash
        </p>
        <p>
          Not to Anything
        </p>
        <input
          class="local"
          id="name"
          name="Doctor"
          placeholder="Enter your Name"
          type="text"
        />
        <br />
        <br />
        <br />
        <img
          alt="prompt.jpg"
          src="https://th.bing.com/th/id/OIP.MPP1fw6m0Gy1xNGZF2ioIAHaFG?rs=1&pid=ImgDetMain"
          title="this is leaf"
        />
        <h1>
          This is Your Name
        </h1>
        <button>
          what is Name
        </button>
        <p />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <p>
        Hey I Am Akash
      </p>
      <p>
        Not to Anything
      </p>
      <input
        class="local"
        id="name"
        name="Doctor"
        placeholder="Enter your Name"
        type="text"
      />
      <br />
      <br />
      <br />
      <img
        alt="prompt.jpg"
        src="https://th.bing.com/th/id/OIP.MPP1fw6m0Gy1xNGZF2ioIAHaFG?rs=1&pid=ImgDetMain"
        title="this is leaf"
      />
      <h1>
        This is Your Name
      </h1>
      <button>
        what is Name
      </button>
      <p />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
